ui/packages/catalog-ui-search/src/main/webapp/component/workspace-interactions/workspace-interactions.view.js

```diff
@@ -50,27 +48,26 @@ module.exports = Marionette.ItemView.extend({
   ui: {},
   initialize: function() {},
   onRender: function() {
-    this.checkIfSubscribed()
-    this.handleLocal()
-    this.handleShareable()
-  },
-  handleLocal: function() {
-    this.$el.toggleClass('is-local', this.model.isLocal())
-  },
-  handleShareable: function() {
-    const notShareable = function(that) {
-      const userLogin = user.get('user').get('email')
-      if (that.model.get('metacard.owner') === userLogin) {
-        return false
-      }
-      const accessAdministrators =
-        that.model.get('security.access-administrators') || []
-      return !accessAdministrators.includes(userLogin)
-    }
-    this.$el.toggleClass('is-not-shareable', notShareable(this))
-  },
-  checkIfSubscribed: function() {
     this.$el.toggleClass('is-subscribed', Boolean(this.model.get('subscribed')))
+    this.$el.toggleClass('is-local', this.model.isLocal())
```

Comment:
✏️ 	I think using the simple helper methods was more readable than inlining them here